### PR TITLE
Mobile prevent scroll

### PR DESCRIPTION
**Pull request by pgiraud**
Also closes side navigation when swiping the map.

### CODE REVIEW

**Comment by pgiraud** [comment]
I have a problem that I don't understand for now. The ngTouch directives don't seem to be taken into account in the hosted examples.

**Comment by pgiraud** [comment]
Fixes #399.

**Comment by sbrunner** [comment]
Looks good :-)

**Comment by pgiraud** [comment]
Any idea about why ngTouch directives are not taken into account in built mode?

**Comment by sbrunner** [comment]
missing `goog.require('ngTouch');`?

File: contribs/gmf/apps/appmodule.js [hunk 1]
```diff
@@ -15,4 +15,4 @@ goog.require('gmf');
 /**
  * @type {!angular.Module}
  */
-var appModule = angular.module('app', [gmfModule.name]);
+var appModule = angular.module('app', [gmfModule.name, 'ngTouch']);
```
Comment (sbrunner) [inline contribs/gmf/apps/appmodule.js]: What does it mean?

**Comment by pgiraud** [comment]
I prefer leaving the swipe gesture besides for now.
I'll open a new issue for that.

**Pull request merged**